Classes/Console/Command/BackendCommandController.php

```diff
@@ -148,8 +148,8 @@ public function createAdminCommand(string $username, string $password)
             $this->outputLine('<warning>Given username "%s" contains invalid characters. Using "%s" instead.</warning>', [$givenUsername, $username]);
         }
 
-        if (strlen($username) < 4) {
-            $this->outputLine('<error>Username must be at least 4 characters.</error>');
+        if (strlen($username) == 0) {
```

Comment:
You should use `empty()` instead now.

Reply:
PHP doc:
empty() is essentially the concise equivalent to !isset($var) || $var == false

I would personally use !$username ;) so please don't ask me for something like empty(strlen(..)) cuz it is crazy.

Reply:
cleanest thing would be `$username === ''` :)

Will push a change reflecting that